packages/in_app_purchase/in_app_purchase_storekit/example/shared/RunnerTests/InAppPurchasePluginTests.m

```diff
@@ -16,6 +16,10 @@ @interface InAppPurchasePluginTest : XCTestCase
 
 @end
 
+@interface SKPaymentTransactionMessage ()
+- (NSArray *)toList;
+@end
```

Comment:
I think you should just copy and paste these methods instead of making the private method visible.  That will protect us from any changes in pigeon.